scripts/new_project.kts

```diff
@@ -57,26 +69,48 @@ object NewProject {
                     val oldDirectory = File(
                         javaDirectory, TEMPLATE_PACKAGE_NAME.replace(
                             oldValue = DOT_SEPARATOR,
-                            newValue = File.separator
+                            newValue = fileSeparator
                         )
                     )
                     val newDirectory = File(
                         javaDirectory, packageName.replace(
                             oldValue = DOT_SEPARATOR,
-                            newValue = File.separator
+                            newValue = fileSeparator
                         )
                     )
 
                     val tempDirectory = File(javaDirectory, "temp_directory")
-                    oldDirectory.copyRecursively(tempDirectory)
+                    copyFiles(fromPath = oldDirectory.absolutePath, toPath = tempDirectory.absolutePath)
                     oldDirectory.parentFile?.parentFile?.deleteRecursively()
                     newDirectory.mkdirs()
-                    tempDirectory.copyRecursively(newDirectory)
+                    copyFiles(fromPath = tempDirectory.absolutePath, toPath = newDirectory.absolutePath)
                     tempDirectory.deleteRecursively()
                 }
         }
     }
 
+    private fun renamePackageNameWithinFiles() {
+        showMessage("=> 🔎 Renaming package name within files...")
+        File(projectPath)
+            ?.walk()
+            .filter { it.isFile && it.name != "debug.keystore" && it.name != "gradle-wrapper.jar"}
```

Comment:
As usual, it got corrupted after rename function. 😅 So I have neglected that file from the method. I think we should only rewrite `.class`, `.kt`, and `.xml` files because whenever we try to rewrite the other types of files, they got corrupted.

Reply:
Hmmm, good point 👍 

Then I wonder if we should change it to: `.filter { it.endsWith(".kt") || it.endsWith(".xml") }` instead

Any reason we should include `.class` too? 🙏

Reply:
Nope. I just explained it as an example 😆